client/components/webcamModule.js

```diff
@@ -0,0 +1,114 @@
+angular.module('in-your-face')
+.controller('webcamModuleCtrl', function($scope) {
+  console.log("123");
```

Comment:
no worries, i was checking coding guidelines. We can change it for next version also. Thanks.